web/impact/impact/v1/views/reserve_office_hour_view.py

```diff
@@ -210,3 +223,52 @@ def _response(self):
             'header': self.header,
             'detail': self.detail,
             'timecard_info': self.timecard_info})
+
+    def get_calendar_data(self, counterpart_name):
+        if self.mentor_recipient:
+            name = self.startup.name if self.startup else counterpart_name
+            title = self.OFFICE_HOUR_TITLE.format(counterpart_name)
+        else:
```

Comment:
No need for this else branch, since it duplicates line 230. (outdent 230 and delete 231-232)

Reply:
Had forgotten something here

Reply:
The original comment still stands. line 232 duplicates line 230.

Reply:
this has been updated

Reply:
sorry, I noticed that I deleted your second comment instead of mine. I have fixed the issue. thanks